src/actions/Ability.js

```diff
@@ -0,0 +1,11 @@
+export class Ability {
```

Comment:
What is the purpose of this class, it doesn't have logic, again, use a linter and another tool to format your code before you push.